release/public-v1:  Add FAQ chapter with the following questions: (#157)
ufs-community/ufs-weather-model: doc/UsersGuide/source/FAQ.rst

.. _FAQ:

***
FAQ
***

==============================================================
How do I build and run a single test of the UFS Weather Model?
==============================================================

An efficient way to build and run the UFS Weather Model is to use the regression test
(``rt.sh``).  This script is widely used by model developers on Tier 1 and 2 platforms
and is described in :numref:`Section %s <ConductingRegTests>`.  The advantages to this approach are:

> - You can bypass the workflow, pre- and post-processing steps.

Reviewer: I suggest changing the language here. The UFS WM does not have a workflow with pre- and post- that needs to be bypassed (the App does, but this documentation is for the model).
```suggestion
- It does not require a workflow, pre- or post-processing steps.
```

         RTPWD=${RTPWD:-$DISKNM/ufs-public-release-20200224/${COMPILER^^}}

4. Run the ``rt.sh`` script from the ``tests`` directory:

   .. code-block:: console

       ./rt.sh -k -l my_test.conf >& my_test.out &

   Check ``my_test.out`` for build and run status, plus other standard output. Check
   ``/glade/scratch/$USER/FV3_RT/rt_PID`` for the model run, where ``PID`` is a process ID.
   The build will take about 10-15 minutes and the run will be fast, depending on how long
   it waits in the queue.  A message ``"REGRESSION TEST WAS SUCCESSFUL"`` will be written to this
   file, along with other entertainment: ``'Elapsed time: 00h:14m:12s. Have a nice day!'``.

5. When the build and run are complete, modify the namelist or ``model_configure`` files
   and re-run by submitting the ``job_card`` file:

   .. code-block:: console

       qsub job_card

============================================
How do I change the length of the model run?
============================================
In your run directory, there is a file named ``model_configure``.  Change the
variable ``nhours_fcst`` to the desired number of hours.

=====================================================

> How do I change the model output to netcdf or nemsio?

Reviewer: ```suggestion
How do I select the file format for the model output (NetCDF or NEMSIO)?
```
